app/routes/forum/components/ForumListEntry.tsx

```diff
@@ -0,0 +1,26 @@
+import { Flex } from '@webkom/lego-bricks';
+import cx from 'classnames';
+import { Link } from 'react-router-dom';
+import styles from './ForumList.css';
+import type { PublicForum } from 'app/store/models/Forum';
+
+const ForumListEntry = ({
+  subForum,
+  className,
+}: {
+  subForum: PublicForum;
+  className: string;
+}) => {
+  return (
+    <Flex column className={cx(styles.listEntry, className)}>
+      <Link to={`/forum/${subForum.id}`}>
+        <h2>{subForum.title}</h2>
+        <div>
+          <h4>{subForum.description}</h4>
+        </div>
+      </Link>
```

Comment:
See other comment about the thread entries. I like that these are links, but not a fan of all the text being red.